metrics/system_metrics.go

```diff
@@ -0,0 +1,42 @@
+package metrics
+
+import (
+	"runtime"
+
+	"github.com/rcrowley/go-metrics"
+)
+
+const allocGauge = "runtime.mem.bytes_allocated_and_not_yet_freed"
+const heapObjectsGauge = "runtime.mem.total_number_of_allocated_objects"
+const totalPauseGauge = "runtime.mem.pause_total_ns"
+const lastPauseGauge = "runtime.mem.last_pause"
+
+func collectSystemMetrics() (err error) {
```

Comment:
You can use https://godoc.org/github.com/rcrowley/go-metrics#RegisterRuntimeMemStats instead of this